javascript-carousel/main.js

```diff
@@ -0,0 +1,97 @@
+var $imageButton1 = document.querySelector('.image1');
+var $imageButton2 = document.querySelector('.image2');
+var $imageButton3 = document.querySelector('.image3');
+var $imageButton4 = document.querySelector('.image4');
+var $imageButton5 = document.querySelector('.image5');
+var $arrowLeft = document.querySelector('.arrow-left');
+var $arrowRight = document.querySelector('.arrow-right');
+var $image = document.querySelector('.image');
+
+function imageOneHandle(event) {
+  $image.setAttribute('src', 'images/025.png');
+  $imageButton1.className = 'image1 fa-solid fa-face-grin-stars fa-xl';
+  $imageButton2.className = 'image2 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton3.className = 'image3 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton4.className = 'image4 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton5.className = 'image5 fa-regular fa-face-grin-stars fa-xl';
+}
+
+$imageButton1.addEventListener('click', imageOneHandle);
+
+function imageTwoHandle(event) {
+  $image.setAttribute('src', 'images/001.png');
+  $imageButton1.className = 'image1 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton2.className = 'image2 fa-solid fa-face-grin-stars fa-xl';
+  $imageButton3.className = 'image3 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton4.className = 'image4 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton5.className = 'image5 fa-regular fa-face-grin-stars fa-xl';
+}
+
+$imageButton2.addEventListener('click', imageTwoHandle);
+
+function imageThreeHandle(event) {
+  $image.setAttribute('src', 'images/004.png');
+  $imageButton1.className = 'image1 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton2.className = 'image2 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton3.className = 'image3 fa-solid fa-face-grin-stars fa-xl';
+  $imageButton4.className = 'image4 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton5.className = 'image5 fa-regular fa-face-grin-stars fa-xl';
+}
+
+$imageButton3.addEventListener('click', imageThreeHandle);
+
+function imageFourHandle(event) {
+  $image.setAttribute('src', 'images/007.png');
+  $imageButton1.className = 'image1 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton2.className = 'image2 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton3.className = 'image3 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton4.className = 'image4 fa-solid fa-face-grin-stars fa-xl';
+  $imageButton5.className = 'image5 fa-regular fa-face-grin-stars fa-xl';
+}
+
+$imageButton4.addEventListener('click', imageFourHandle);
+
+function imageFiveHandle(event) {
+  $image.setAttribute('src', 'images/039.png');
+  $imageButton1.className = 'image1 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton2.className = 'image2 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton3.className = 'image3 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton4.className = 'image4 fa-regular fa-face-grin-stars fa-xl';
+  $imageButton5.className = 'image5 fa-solid fa-face-grin-stars fa-xl';
+}
```

Comment:
You should try to think of a way to do this that doesn't require an individual function to change to each image. If you ever wanted to add another image, you would need to revise _all_ of your code, that sounds like a nightmare 💀. I recommend you think about how you can keep track of the list of images in order, as well as the image currently being shown using variables.

Reply:
https://user-images.githubusercontent.com/101234537/181681612-708ab323-bae6-4c02-b5cb-7f700d452911.mov